packages/grafana-ui/src/utils/processTableData.ts

```diff
@@ -0,0 +1,133 @@
+import { TableData, Column } from '../types/index';
+
+import Papa, { ParseError, ParseMeta } from 'papaparse';
+
+// Subset of all parse options
+export interface ParseConfig {
```

Comment:
ParseTableOptions ?  (since this is export from @grafana/ui we need to be specific with naming

Reply:
👍